proc_scripts/plotting.py

```diff
@@ -68,12 +68,24 @@ class fl_mod(figlist_var):
     plotname:   string with name of plot 
     s:          nddata being analyzed
     thisrange:  range along x axis to be analyzed
+    f:          fitdata 
+                data (a fitting instance), on which `f.fit()` has already been run
+    guess:      None or nddata
+                The result of `s.settoguess();s.eval(100)` where 100 can be
+                any integer.
+                Used to display the guess on the plot as well.
+    name:       str
+                the name of the plot
```

Comment:
As noted elsewhere, this was moved and should not have been

Reply:
added back in